### ISSUE TITLE
Error when calculating GSVA based on UCell method

### ISSUE THREAD

**Issue by YuxinYin0906** [guokai8/scGSVA#28]
Hi,

Thank you for making this amazing package! It has been a life saver for me.
However, I ran into a problem when trying to calculate gsva results using the method "UCell". 
` #annotation
  hsko<-buildAnnot(species="mouse",keytype="SYMBOL",anntype="GO")
  
  # result based on ssgsea
  res<-scgsva(scRNA_all_int,hsko,method="UCell") ## or use UCell`

## The error message is as followed: 
"Error in calculate_Uscore(m, features = features, maxRank = maxRank, chunk.size = chunk.size,  : 
  One or more signatures contain more genes than maxRank parameter.
            Increase maxRank parameter or make shorter signatures
Calls: scgsva".

I tried to change the anntype to "KEGG" and the function works smoothly, so I guess it's due to the fact that GO is a more enriched dataset than KEGG.
I understand that this might be more related to the UCell package, but I wonder if there's any way we could set the parameter for `maxrank` in the scgsva() function.

**Comment by guokai8** [comment]
Hi @YuxinYin0906,
I will take a look at it and try to fix it soon. I guess you are right.
K

**Comment by merckey** [comment]
I think you can add one additional parameter from UCell `maxRank`.

`res<-scgsva(scRNA_all_int,hsko,method="UCell", maxRank = 3000)`

**Comment by guokai8** [comment]
@merckey 
Yes. I will add this, and you can directly add the "maxRank" when you run it now.
Thanks!
K
